common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala

```diff
@@ -38,4 +42,15 @@ object CollectionsManager {
 
   // We could use `ValidationNel`s here, but that's overkill
   def isValidPathBit(s: String) = if (s.contains(delimiter)) false else true
+
+  val collectionColours = Map(
+    "g2"           -> "#000000",
+    "observer"     -> "#006f94",
+    "culture"      -> "#d1008b",
+    "film & music" -> "#b1532f"
+  )
+
+  def getCollectionColour(s: String) = collectionColours.get(s)
+
+  def getCssColour(path: List[String]): String = path.headOption.flatMap(getCollectionColour).getOrElse("#cccccc")
```

Comment:
I prefer to keep the client stupid - if we make this configurable later, having it on the client would be annoying.